end-to-end/cypress/e2e/portal/publication-page.cy.js

```diff
@@ -0,0 +1,110 @@
+// This test was written against a existing publication in the dev/test env and will need to be updated if that publication is deleted
+
+const publicationId = "cd03fa15322492f4173cdbdc30c74a62";
+const title = "sdarfg";
+
+describe("Publication page", () => {
+  context("macbook-size", () => {
+    beforeEach(() => {
+      cy.viewport("macbook-15");
+      cy.visit(`/browse/publication/${publicationId}`);
+    });
+    it("has a title, abstract, manuscript link, citation, data types, organs, and publication date listed", () => {
+      cy.findByTestId("entity-title").contains(title);
+      cy.findByTestId("publication-abstract").should("exist");
+      cy.findByTestId("publication-manuscript-link").should("exist");
+      cy.findByTestId("publication-citation").should("exist");
+      cy.findByTestId("publication-data-types").contains("CODEX");
+      cy.findByTestId("publication-organs").contains("Lymph Node");
+      cy.findByTestId("publication-date").contains("2023-02-27");
+    });
+    it("has a table of contents with links to the publication summary, data, files, authors, and provenance", () => {
+      cy.findByTestId("table-of-contents")
+        .should("contain", "Summary")
+        .and("contain", "Data")
+        .and("contain", "Files")
+        .and("contain", "Authors")
+        .and("contain", "Provenance");
+    });
+    it('has a "Data" section with tabs for tables of donors, samples, and datasets', () => {
+      // Donors tab is active by default
+      cy.findByTestId("donors-tab")
+        .should("exist")
+        .and("contain", "Donors")
+        .and("contain", "(1)");
+      cy.findAllByTestId("donor-row").should("have.length", 1);
+
+      // Samples tab needs to be clicked to activate
+      cy.findByTestId("samples-tab")
+        .should("exist")
+        .and("contain", "Samples")
+        .and("contain", "(3)")
+        .click();
+      cy.wait(100);
+      cy.findAllByTestId("sample-row").should("have.length", 3);
```

Comment:
`find` will retry if the element isn't present in the DOM with a default timeout of [4000ms](https://docs.cypress.io/guides/references/configuration#Timeouts). Is `wait` necessary here? If so you can extend the timeout of `find`.

Reply:
When I was running this locally, any tests which relied on the page state updating from clicking on a tab failed unless I explicitly added a short wait period. I tried to work around this by playing with the timeout, by chaining with `.then`, and trying the `cypress-real-events` library which previously solved a similar issue for me. Unfortunately, the only approach which worked for me at the time was adding this `.wait`.